pkg/cmd/release/attestation/options.go

```diff
@@ -0,0 +1,76 @@
+package attestation
+
+import (
+	"fmt"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/internal/ghinstance"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/api"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/io"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/verification"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+)
+
+const ReleasePredicateType = "https://in-toto.io/attestation/release/v0.1"
+
+type AttestOptions struct {
+	Config           func() (gh.Config, error)
+	HttpClient       *http.Client
```

Comment:
currently have both HTTPClient and APIClient to support calls to shared/fetch.go and attestation/api, but will plan to refactor them into a single client.